t/04-openresty/ffi/shm/020-metrics_define.t

```diff
@@ -20,10 +20,13 @@ __DATA__
         access_by_lua_block {
             local shm = require "resty.wasmx.shm"
 
+            local bins = { 1, 3, 5 }
```

Comment:
Let's also add a test with unsorted bins - unless I missed it?

Reply:
Good point. As privately discussed, we are forbidding unsorted bins.